src/System.Management.Automation/engine/remoting/commands/PushRunspaceCommand.cs

```diff
@@ -172,6 +174,18 @@ public override PSCredential Credential
                    ParameterSetName = EnterPSSessionCommand.VMNameParameterSet)]
         public String ConfigurationName { get; set; }
 
+        #region Supress PSRemotingBaseCmdlet SSH hash parameter set
```

Comment:
Fixed.  This cmdlet derives from a base cmdlet that includes the SSHConnection parameter.  But this cmdlet cannot support SSHConnection hashtable (since Enter-PSSession works only on single session) and we have to remove it from the parameter set.  This is common in our code base and the word "suppress" is used for this and I continued using the term here.